src/Microsoft.AspNetCore.Rewrite/Internal/ModRewrite/TestStringParser.cs

```diff
@@ -97,7 +96,7 @@ public static Pattern Parse(string testString)
         /// <param name="context">The ParserContext</param>
         /// <param name="results">The List of results which the new condition parameter will be added.</param>
         /// <returns>true </returns>
-        private static void ParseConditionParameter(ParserContext context, List<PatternSegment> results)
+        private void ParseConditionParameter(ParserContext context, List<PatternSegment> results)
```

Comment:
Comments are good. Don't delete them. I frequently amazed by my own comments which I left usually not long ago ......